src/actions/referralsActions.js

```diff
@@ -330,12 +331,27 @@ export const fetchReferralRewardAction = () => {
       type: FETCHING_REFERRAL_REWARD_AMOUNT,
     });
 
-    const referralRewards: RewardsByCompany = await api.getReferralRewardValue(walletId, referralToken);
+    const referralCampaignsInfo = await api.getReferralCampaignsInfo(walletId, referralToken);
+    const referralRewards: RewardsByCompany = Object.keys(referralCampaignsInfo)
+      .reduce((rewardsByCampaign, campaignName) => {
+        const isCampaignActive = get(referralCampaignsInfo, `[${campaignName}].isActive`);
```

Comment:
I think there is no need for those brackets (I could be wrong)
```suggestion
        const isCampaignActive = get(referralCampaignsInfo, `${campaignName}.isActive`);
```

Reply:
On the other hand, if `campaignName` could have spaces, then my solution won't work